Assets/SteamVR_Unity_Toolkit/Scripts/Archery/Quiver.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Quiver : MonoBehaviour {
+
+    BowAim Bow;
+    public GameObject ArrowPrefab;
+
+    public GameObject VRHead;
+
+    float internalCD = 0.15f;
+    float cd;
+
+    void Start()
+    {
+        Bow = GetComponent<BowAim>();
+        if (VRHead == null)
+            VRHead = FindObjectOfType<SteamVR_GameView>().gameObject;
```

Comment:
Finding the headset using SteamVR_GameView doesn't work in Unity 5.4 because it's been deprecated.

Touchpad_Walking.cs has the same issue so this is how I resolved it:

https://github.com/thestonefox/SteamVR_Unity_Toolkit/blob/master/Assets/SteamVR_Unity_Toolkit/Scripts/SteamVR_TouchpadWalking.cs#L37-L41

Same should work here.